NetKAN/TweakScaleCompanion.netkan

```diff
@@ -0,0 +1,31 @@
+spec_version: v1.8
+identifier: TweakScaleCompanion
+name: TweakScale Companion
+abstract: >-
+  **TweakScale Companion Program** is a series of "Add'On's Add'Ons",
+  adding TweakScale /L support for third parties but decoupling the
+  target Add'On's life cycle from TweakScale's, easing maintenance
+  and distribution efforts. This package aims to make TweakScale
+  user's life easier at first installs with a (most of the time)
+  updated collection with all the currently available Companions
+  at once.
+author:
+  - LisiasT
+$kref: '#/ckan/spacedock/3202'
+$vref: '#/ckan/ksp-avc/TweakScaleCompanion.version'
+x_netkan_force_v: true
+license:
+  - restricted
+release_status: testing
+tags:
+  - plugin
+  - convenience
+install:
+  - file: GameData/TweakScaleCompanion
+    install_to: GameData
```

Comment:
```suggestion
```

You don't need this; the default install stanza when it's left out is to `find` the identifier and install it to GameData, which is equivalent to this.

Reply:
Using default CKAN behaviours instead of specifying it